packages/vite/src/client/client.ts

```diff
@@ -126,6 +126,7 @@ function cleanUrl(pathname: string): string {
 }
 
 let isFirstUpdate = true
+const invalidLinkTags = new Set<HTMLLinkElement>()
```

Comment:
Maybe we could use `WeakSet` to be safe?

Reply:
Hmm..., I'm not sure about the client code's browser support

Reply:
We are transpiling client code to es2019. So I believe it's fine.